[2020-07-31 03:48:04,750] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_jira.load_issues 2020-07-31T03:47:00+00:00 [queued]>
[2020-07-31 03:48:04,789] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_jira.load_issues 2020-07-31T03:47:00+00:00 [queued]>
[2020-07-31 03:48:04,790] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-07-31 03:48:04,790] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2020-07-31 03:48:04,790] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-07-31 03:48:04,822] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): load_issues> on 2020-07-31T03:47:00+00:00
[2020-07-31 03:48:04,830] {standard_task_runner.py:53} INFO - Started process 22457 to run task
[2020-07-31 03:48:04,958] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_jira.load_issues 2020-07-31T03:47:00+00:00 [running]> 02f6ef7ded4c
[2020-07-31 03:48:05,059] {logging_mixin.py:112} INFO - [2020-07-31 03:48:05,059] {base_hook.py:84} INFO - Using connection to: id: postgres_jira. Host: 95.217.217.10, Port: 54320, Schema: jira, Login: joeexotic, Password: XXXXXXXX, extra: None
[2020-07-31 03:48:05,087] {logging_mixin.py:112} INFO - Insertion took 0:00:00.017476 seconds
[2020-07-31 03:48:05,091] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-07-31 03:48:05,119] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_jira, task_id=load_issues, execution_date=20200731T034700, start_date=20200731T034804, end_date=20200731T034805
[2020-07-31 03:48:14,723] {logging_mixin.py:112} INFO - [2020-07-31 03:48:14,722] {local_task_job.py:103} INFO - Task exited with return code 0
